### PR TITLE
Added new components (wrappers, form items)

### PR DESCRIPTION
**Pull request by KateVuilova**
Refactoring and adding new components

### CODE REVIEW

File: src/components/SignUpForm/SignUpForm.js [hunk 1]
```diff
@@ -33,50 +26,46 @@ class SignUpForm extends Component {
...
-                                />
-                                <Button text="SIGN UP" type="submit" className="accept-button" />
-                            </form>
-                            <div className="auth-link-wrapper">
-                                <Link to="/login" className="auth-link">
-                                    Back to login
-                                </Link>
-                            </div>
+            <AuthWindowAligner>
+                <ShadowWrapper className="signup-form">
```
Comment (vdanyliv) [inline src/components/SignUpForm/SignUpForm.js]: we don't need to create separate component that just simply add shadow and class to our window, move this functionality to general window, and predict possibility to enable or disable shadow

File: src/components/FormItem/FormItem.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+const FormItem = props => {
+    return (
+        <div className="form-item-wrapper">
+            <input
+                className="form-item"
+                type={props.field.type}
+                placeholder={props.field.placeholder}
+                {...props.field.input}
+                required
```
Comment (vdanyliv) [inline src/components/FormItem/FormItem.js]: if you plan to reuse this component in other part of your application, you need to predict possibility to set it like required or not, also think about class names.

Also you should think what type of field you will use on you project, i suggest to create separate component for each field, and name it like: InputControl.js = <Name><Type>.js

File: src/components/LogInForm/LogInForm.js [hunk 3]
```diff
@@ -5,6 +5,9 @@ import "./LogInForm.css";
 import ShadowWrapper from "../ShadowWrapper";
 import Button from "../Button";
 import Copyright from "../Copyright";
+import AuthWindowAligner from "../AuthWindowAligner";
+import AuthHeader from "../AuthHeader";
+import Formitem from "../FormItem";
```
Comment (vdanyliv) [inline src/components/LogInForm/LogInForm.js]: you misspell name of your component

**Comment by mrmckeb** [comment]
This can be merged @KateVuilova ;)

**Pull request merged**